plugins/invoice-csv-export/src/public.php

```diff
@@ -31,6 +31,16 @@
         'createdDateTo' => $_GET['until'],
     ];
 
+    // make sure the dates are in YYYY-MM-DD format
+    if ($parameters['createdDateFrom']) {
+        $parameters['createdDateFrom'] = new \DateTimeImmutable($parameters['createdDateFrom']);
```

Comment:
Does it make sense to catch+handle an invalid format exception here? Dtto `createdDateTo`, dtto revenueReport

Reply:
Not really, this is basically hotfix for Safari as other browsers have `<input type="date">` implemented and are sending the dates in correct format.
As there is no error handling at all in these plugins, it would have to be extended with that, but that's out of scope here. We could do it in the future if we want to.